modules/core/02-client/proposal_handler.go

```diff
@@ -3,7 +3,7 @@ package client
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
```

Comment:
why not use `v1`?

Reply:
So what's the difference between `v1` and `v1beta1`? In some places we use one and in others the other...

Reply:
Looks like I need `v1beta1` here because `v1` doesn't have `Handler` and `Content`...

Reply:
opened #1282